packages/appstore/src/components/wallet-account/wallet-account.tsx

```diff
@@ -0,0 +1,232 @@
+                                    icon_name='TradingDerived'
+                                    is_app_installed={false}
+                                    button_className='wallet-account__app-launcher-button'
+                                    jurisdiction=''
+                                    description='Trade CFDs on MT5 with synthetic indices that simulate real-world market movements.'
+                                />
+
+                                <AppsLauncher
+                                    app_name='Financial'
+                                    icon_name='TradingFinancial'
+                                    is_app_installed={false}
+                                    button_className='wallet-account__app-launcher-button'
+                                    jurisdiction=''
+                                    description='Trade CFDs on MT5 with forex, stocks &amp; indices, commodities, and cryptocurrencies.'
+                                />
+                            </div>
+                        </div>
+
+                        <div className='wallet-account__apps-wrapper'>
+                            <div className='wallet-account__apps-title no-mt'>
+                                <Text bold type='paragraph-1'>
+                                    {localize('Deriv X')}
+                                </Text>
+                            </div>
+                            <div className='wallet-account__apps'>
+                                <AppsLauncher
+                                    app_name='Deriv X2'
+                                    icon_name='DerivX'
+                                    is_app_installed={false}
+                                    button_className='wallet-account__app-launcher-button'
+                                    jurisdiction=''
+                                    handleClick={wallet_store.installDerivApps}
+                                    description='Trade CFDs on Deriv X with synthetic indices that simulate real-world market movements.'
+                                />
+                            </div>
+                        </div>
+
+                        <div>
+                            <Text bold type='subtitle-1'>
+                                {localize('Options & Multipliers')}
+                            </Text>
+                            <Text bold={false} type='paragraph-1'>
+                                {localize('Earn fixed payouts by predicting price movements with ')}
+                                <a href='#' className='link-text'>
+                                    {localize('Options')}
+                                </a>
+                                {localize(', or combine the upside of CFDs with the simplicity of Options with ')}
+
+                                <a href='#' className='link-text'>
+                                    {localize('Multipliers')}
+                                </a>
```

Comment:
```suggestion
<Localize 
  18n_default_text=‘Earn fixed payouts by predicting price movements with 
  <0>Options </0>, or combine the upside of CFDs with the simplicity of Options with 
  <1>Multipliers</1>‘ 
  components={[
    <a key={0} href='#' className='link-text' />
    <a key={1} href='#' className='link-text' />   
  ]} />
```
Are we have any chance to use this or something like this? 🙏  As far as I know current approach is bad for translation